packages/driver/cypress/e2e/dom/elements.cy.ts

```diff
@@ -321,4 +321,14 @@ describe('src/dom/elements', () => {
       expect(elementFromPoint(doc, 1, 2)).to.equal(shadowHostNode)
     })
   })
+
+  context('.isFocusedOrInFocused', () => {
+    it('should traverse shadow roots when determining if an element has focus', () => {
+      cy.visit('/fixtures/shadow-dom-focus.html')
```

Comment:
Is it possible to use [`/fixtures/shadow-dom-type.html`](https://github.com/cypress-io/cypress/blob/develop/packages/driver/cypress/fixtures/shadow-dom-type.html) to verify this behavior? Would limit the need for an additional fixture.

Reply:
Updated.